Sources/TootSDK/TootClient/TootClient+Media.swift

```diff
@@ -42,6 +58,16 @@ extension TootClient {
     /// - Parameter id: The local ID of the attachment.
     /// - Returns: `Attachment` with a `url` to the media if available. `nil` otherwise.
     public func getMedia(id: String) async throws -> MediaAttachment? {
+        let response = try? await getMediaRaw(id: id)
```

Comment:
I don't think ignoring errors here is correct. A better approach would be to move the special mastodon case with status code 206 to this function. The raw function then could be simplified to use same logic like other api call without any special handling.